src/stdlib/GlobalUtilities.ts

```diff
@@ -39,3 +39,23 @@ export const GetInterface = new Callable("GetInterface", {
         return object.interfaces.get(ifname.value.toLowerCase()) || BrsInvalid.Instance;
     },
 });
+
+export const FindMemberFunction = new Callable("FindMemberFunction", {
+    signature: {
+        args: [
+            new StdlibArgument("object", ValueKind.Object),
+            new StdlibArgument("funname", ValueKind.String),
```

Comment:
Nit, but can we camelcase uses of _funname_ as is done in the Roku docs?

https://developer.roku.com/en-ca/docs/references/brightscript/language/global-utility-functions.md#findmemberfunctionobject-as-object-funname-as-string-as-interface

Reply:
I thought of that but wanted to keep things consistent, if you see `GetInterface` above we do the same for `ifname` arg. Also in `RoSGNode.ts` all of the arguments are in lowercase (i.e. `fieldname`, `alwaysnotify`, `functionname`, etc) I think we should keep it as it right now to avoid changing all other occurrences

Reply:
Agreed — this might be part of a larger validation pass where we ensure our capitalization is correct across the codebase.  It's a "fit and finish" type of issue that we'll definitely need to handle eventually!